test/tests.js

```diff
@@ -5,6 +5,6 @@ require('test/mouse_move_test');
 require('test/mouse_button_test');
 //require('test/mouse_regressions_test');
 require('test/drag_test_basic');
-require('test/drag_test_offsets');
+//require('test/drag_test_offsets');
```

Comment:
Are these tests not working anymore?

Reply:
These tests are for an upcoming feature that I am working on next.

Reply:
Ah, this was added when you merged your pending work while making the last release, got it.